samples/sensor/apds9960/README.rst

```diff
@@ -1,52 +1,36 @@
-.. _apds9960-sample:
+.. _apds9960:
 
-RGB and Gesture Sensor
-######################
+APDS9960 RGB, ambient, Gesture Sensor
+#####################################
 
 Overview
 ********
 
-This sample utilizes APDS-9960 Sensor and reads RGB values from the sensor
-then displays the color through the APA102C LED.
+A sensor application that demonstrates how to use the APDS9960 to get ambient light,
+RGB and proximity data.
 
+Building and Running
+********************
 
-.. note::
-   This sample does not use the Zephyr sensor APIs
-
-Wiring
-******
-
-The SparkFun RGB and Gesture Sensor was being used:
-
-- https://www.sparkfun.com/products/12787
-- https://www.adafruit.com/product/2343
-
-For APA102C, on the sensor subsystem (ARC) side of Arduino 101:
-
-1. GPIO_SS_2 is on AD0 (for APA102C data)
-2. GPIO_SS_3 is on AD1 (for APA102C clock)
-
-The GPIO driver is being used for bit-banging to control the APA102C LED.
-
-The APA102/C requires 5V data and clock signals, so logic level shifter
-(preferred) or pull-up resistors are needed.  Make sure the pins are 5V
-tolerant if using pull-up resistors.
+samples provide output to the console. It requires an apds9960 sensor and connect with
+Arduino101. You can see different intesity and proximity output.
 
-.. important::
+.. code-block:: console
 
-   The APA102C are very bright even at low settings.  Protect your eyes
-   and do not look directly into those LEDs.
+        $ cd samples/sensors/apds9960
+        $ make BOARD=arduino_101_sss flash
 
-Building and Running
-********************
+        $ cd ../../..
+        $ cd tests/booting/stub
+        $ make BOARD=arduino_101 flash
 
-This sample can be built for multiple boards, in this example we will build it
-for the Arduino 101 board:
+Sample Output
+=============
 
 .. code-block:: console
 
-   $ cd samples/sensors/apds9960
-   $ make BOARD=arduino_101_sss
-   $ make BOARD=arduino_101_sss flash # with JTAG
-   .. or
-   $ dfu-util -a sensor_core -D outdir/arduino_101_sss/zephyr.bin # with DFU
+        ambient light intensity without trigger is 387
```

Comment:
Without trigger means without using interrupt in this case. I have already wrote in description that this sample code is just for polling mode. and of course polling means it will read sensor data every second.